aspnet/data/entity-framework-6.rst

```diff
@@ -1,110 +1,97 @@
-Getting Started with ASP.NET Core and Entity Framework 6
+﻿Getting Started with ASP.NET Core and Entity Framework 6
 ===========================================================
 
-By `Paweł Grudzień`_ and `Damien Pontifex`_
+By `Paweł Grudzień`_, `Damien Pontifex`_, and `Tom Dykstra`_
 
 This article will show you how to use Entity Framework 6 inside an ASP.NET Core application.
 
 .. contents:: Sections:
   :local:
   :depth: 1
     
-Prerequisites
--------------
-    
-Before you start, make sure that you compile against full .NET Framework in your project.json as Entity Framework 6 does not support .NET Core. If you need cross platform features you will need to upgrade to `Entity Framework Core`_.
```

Comment:
I think the bit about EF 6 not supporting .NET Core 1.0 is still relevant. Did it get moved somewhere else?